### PR TITLE
Add remaning recurrence fields to task new form

### PR DESCRIPTION
**Pull request by nogates**
#### Description

Adds the `recurrenceMatch` field, which it should be hidden unless the `recurrence` of the task is monthly

### CC
@atd

### CODE REVIEW

File: frontend/app/models/recurrence-match-days.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default [
+  Ember.Object.create({ value: 0, label: "Monday" }),
```
Comment (atd) [inline frontend/app/models/recurrence-match-days.js]: @nogates please note that in Ruby's date of the week `0 == 7 == "Sunday"`

I am not sure if this code is wired to the API call

File: frontend/app/models/recurrence-match-positions.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default [
+  Ember.Object.create({ value: 0, label: "First" }),
+  Ember.Object.create({ value: 1, label: "Second" }),
+  Ember.Object.create({ value: 2, label: "Third" }),
+  Ember.Object.create({ value: 3, label: "Fourth" }),
+  Ember.Object.create({ value: 4, label: "Last" })
```
Comment (atd) [inline frontend/app/models/recurrence-match-positions.js]: Equivalent here, we are using in the server recurrence_match `-1 => "Last"`

I would also include a `-2 => "Next to last"` option

File: frontend/app/models/recurrence-options.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+export default [
+  Ember.Object.create({ value: 0, label: "Daily" }),
```
Comment (atd) [inline frontend/app/models/recurrence-options.js:4]: BTW i18n!!

File: frontend/app/models/recurrence-match-days.js (hunk 1 above)
Comment (nogates) [reply frontend/app/models/recurrence-match-days.js]: It's! I shall change it

File: frontend/app/models/recurrence-match-positions.js (hunk 2 above)
Comment (nogates) [reply frontend/app/models/recurrence-match-positions.js]: :+1:

File: frontend/app/models/recurrence-options.js (hunk 3 above)
Comment (nogates) [reply frontend/app/models/recurrence-options.js:4]: I might do that in a separate PR if you don't mind. this one is getting too big!

Comment (atd) [reply frontend/app/models/recurrence-options.js:4]: No problem

**Review by atd — APPROVED** [review]
:shipit:

**Pull request merged**